charts/actions-runner-controller/README.md

```diff
@@ -158,7 +158,8 @@ All additional docs are kept in the `docs/` folder, this README is solely for do
 | `actionsMetrics.serviceMonitor.timeout`                   | Configure the timeout the timeout of Prometheus scrapping.                                                                                | 30s                                                                                             |
 | `actionsMetrics.serviceAnnotations`                       | Set annotations for the provisioned actions metrics service resource                                                                      |                                                                                                 |
 | `actionsMetrics.port`                                     | Set port of actions metrics service                                                                                                       | 8443                                                                                            |
-| `actionsMetrics.proxy.enabled`                            | Deploy kube-rbac-proxy container in controller pod                                                                                        | true                                                                                            |
+| `actionsMetrics.proxy.enabled`                            | Deploy kube-rbac-proxy container in the actions-metrics-server pod                                                                        | true                                                                                            |
 | `actionsMetrics.proxy.image.repository`                   | The "repository/image" of the kube-proxy container                                                                                        | quay.io/brancz/kube-rbac-proxy                                                                  |
 | `actionsMetrics.proxy.image.tag`                          | The tag of the kube-proxy image to use when pulling the container                                                                         | v0.13.1                                                                                         |
+| `actionsMetrics.proxy.resources`                          | Set the kube-rbac-proxy container resources                                                                                               |                                                                                                 |
```

Comment:
I didn't add `metrics.proxy.resources` to the table because none of the `metrics.*` values are here. Let me know if I should add them.